docs/keep-a-changelog.rst

```diff
@@ -0,0 +1,162 @@
+How to Keep a Changelog in Markdown
+===================================
+
+`Keep a Changelog <https://keepachangelog.com/>`_ is a standardized way to format a news file in `Markdown <https://en.wikipedia.org/wiki/Markdown>`_.
+
+This guide shows you how to configure ``towncrier`` for keeping a Markdown-based news file of a project without using any Python-specific features.
+Everything used here can be use with any other language or platform.
+
+This guide makes the following assumptions:
+
+- The project lives at https://github.com/twisted/my-project/.
+- The news file name is ``CHANGELOG.md``.
+- You store the news fragments in the ``changelog.d`` directory at the root of the project.
+
+Put the following into your ``pyproject.toml`` or ``towncrier.toml``:
+
+.. code-block:: toml
+
+   [tool.towncrier]
+   directory = "changelog.d"
+   filename = "CHANGELOG.md"
+   start_string = "<!-- towncrier release notes start -->\n"
+   underlines = ["", "", ""]
+   template = "changelog.d/changelog_template.jinja"
```

Comment:
For now is ok.
I expect that Markdown is a common use case, so we might want to have a predefined markdown template, that is maintained by the main dev team.